include/circt/Dialect/SV/SVPasses.td

```diff
@@ -173,4 +173,17 @@ def HWExportModuleHierarchy : Pass<"hw-export-module-hierarchy",
   let dependentDialects = ["circt::sv::SVDialect"];
 }
 
+def HWRaiseInOutPorts : Pass<"hw-raise-inout-ports",
```

Comment:
Shouldn't this be called something like "LowerInOutPorts"? Or `EliminateInOutPorts`? 'Breakout'? This doesn't seem like something which raises the level of abstraction, which is what the work 'raise' implies to me.

Reply:
I'd favor `Eliminate` here. `Raise` was originally used since i was "raising" the `sv.read/assign` ops upwards in the instance hierarchy.